gulpfile.js

```diff
@@ -40,9 +42,17 @@ exports['test:a11y'] = a11y;
 
 exports.watch = () => {
   watch(
-    ['src/**/*.{ts,tsx}', '!src/test-utils/**/*.ts', '!**/__tests__/**', '!**/__integ__/**', '!**/__a11y__/**'],
+    [
+      'src/**/*.{ts,tsx}',
+      '!src/test-utils/**/*.ts',
+      '!**/__tests__/**',
+      '!**/__integ__/**',
+      '!**/__a11y__/**',
+      '!src/internal/vendor/**/*.ts',
+    ],
     typescript
   );
   watch(['src/test-utils/dom/**/*.ts', '!src/test-utils/dom/index.ts'], testUtils);
   watch(['style-dictionary/**/*.ts', 'src/**/*.scss'], styles);
+  watch(['src/internal/vendor/**/*.ts', '!src/internal/vendor/**/__tests__/**'], series(typescript, bundleVendorFiles));
```

Comment:
why do we watch the vendor files at all?

Reply:
Because we want to compile and bundle the changes and have them reflected in the lib folder (related to the answer on your comment about quick-build above).

The alternative would be to generate the bundled dependency in a separate step which is not part of watch or quick build. With that we'd get feedback about any issues quiet late in our development workflow.

Reply:
My expectation was that the vendor deps can be only bundled once. Would it be possible for example to create a custom build step that only runs as an npm post-install hook?

Reply:
Is there value in running the bundling with every build?

Reply:
Yes, the bundled artifacts are stored in the `lib` folder which gets recreated every build.
I was exploring the option for having a one time bundling in place, but decided to not move in that direction.

Having a one time bundle, would mean we need to store the bundled JS files inside our src folder (or somewhere outside the lib folder) and we need to move them into the lib folder when building the components as part of the build process. I think the current approach is simpler compared to one-time bundling.

With this approach, we ensure to bundle the latest version into /lib, and consume the latest updates licence information.

What alternative would you suggest to not bundle with every build?